src/infrastructure/idAuthentication/DokobitAuthenticationProvider.test.ts

```diff
@@ -0,0 +1,64 @@
+import { DokobitAuthenticationProvider } from './DokobitAuthenticationProvider';
+import { DokobitClient } from './DokobitClient';
+import { AuthenticationError } from '../../application/service/authentication/AuthenticationError';
+import { AuthenticationSessionToken } from '../../domain/model/idAuthentication/models';
+
+describe('Dokobit authentication provider', () => {
+  it('creates sessions', async () => {
+    const provider = new DokobitAuthenticationProvider({
+      async createSession({ returnUrl }) {
+        if (returnUrl === 'http://localhost:3000/authenticate') {
```

Comment:
Do we need to tie this to the port?

Reply:
This is the url configured in the configuration, i wouldn't do special treatment of it here